README.md

```diff
@@ -1 +1,54 @@
-# rudder-ingester
+# rudder-load
+
+This repository is used to generate an artificial load to test the RudderStack architecture under intense traffic.
+
+## Usage
+
+The service is designed to be run in a Kubernetes cluster.
+
+It is possible to create multiple replicas of the service to increase the load.
+
+Each replica owns a specific writeKey and will send events via multiple go routines. Each go routine will send events
+owning a set of keys. This way we can guarantee in-order delivery of events per writeKey-userId.
```

Comment:
Is event ordering a requirement for load tester ?

Reply:
If you need it is there. If you don't need it just pass the same write key over and over :man_shrugging: